scribble-lib/scribble/private/manual-bind.rkt

```diff
@@ -71,15 +68,13 @@
      (if (and from (pair? from))
          (make-element
           (intern-hover-style
-           (string-append
-            "Provided from: "
-            (string-join (map ~s from) ", ")
-            (let ([from-pkgs (resolve-get/tentative p ri '(exporting-packages #f))])
-              (if (and from-pkgs (pair? from-pkgs))
-                  (string-append
-                   " | Package: "
-                   (string-join (map ~a from-pkgs) ", "))
-                  ""))))
+           (string-join ", "
```

Comment:
@jackfirth I don't think this is correct. The first argument to `string-join` must be a list.